subsquid/src/bondedFinanceProcessor.ts

```diff
@@ -22,18 +23,18 @@ interface NewBondEvent {
  * @param event
  */
 function getNewBondEvent(event: BondedFinanceNewBondEvent): NewBondEvent {
-  const { offerId, nbOfBonds } = event.asV2401 ?? event.asLatest;
+  const { offerId, nbOfBonds } = event.asV2401;
   return { offerId, nbOfBonds };
 }
 
 function getNewOfferEvent(event: BondedFinanceNewOfferEvent): NewOfferEvent {
-  const { offerId, beneficiary } = event.asV2401 ?? event.asLatest;
+  const { offerId, beneficiary } = event.asV2401;
 
   return { offerId, beneficiary };
 }
 
 export async function processNewOfferEvent(
-  ctx: EventHandlerContext,
+  ctx: EventHandlerContext<Store, { event: true }>,
```

Comment:
is this `event: true` needed?

Reply:
I have kept it as a placeholder for future needs:
https://docs.subsquid.io/migrate/migrate-to-fire-squid/#3-change-the-processor-initialization

Not sure what to put as a replacement, happy to do it if you have any suggestion that leads me to the right direction 😄